New package: Fasta2onehot v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: Fasta2onehot
- Repository: https://github.com/kchu25/Fasta2onehot.jl
- Created by: @kchu25
- Version: v1.0.0
- Commit: 6dc9da1037f9411640788d174726fe566c64bb49
- Reviewed by: @kchu25
- Reference: https://github.com/kchu25/Fasta2onehot.jl/commit/6dc9da1037f9411640788d174726fe566c64bb49#commitcomment-147210148
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd601214cad3f0c4bae2833af6096ad3cc5f8e1a2e2397fb807940bcb5e0d87fee5a419981ad13f60dab436fcbd609ade26d168e998962cb9b3e0e87395f4e0cd04406026fce6be1c9c463b7ace601a9dc6d8266e554e70dd0dcd3c8d69e3bd24cfae99f036419dbe3192dc16e6dcee752207d8171f48cc99566744b118a884cc0973fac3408d4a00d6420d34242a08f905d2a5e8b5d805c92c09f78971cb8776ba7b03e1d9dca2796528beb257047e5487 -->

goerz commented:
Thank you for submitting your package! However, please make sure to add some documentation before registering. [At the very least](https://discourse.julialang.org/t/when-does-it-make-sense-to-post-a-new-package-to-the-julia-repository/108701/4), that would be a description of the package's purpose and a small usage example in the README

JuliaRegistrator committed:
New package: Fasta2onehot v1.0.0

UUID: 0c7a3de5-96ef-4511-87db-5c41ebae0efd
Repo: https://github.com/kchu25/Fasta2onehot.jl.git
Tree: 7a4a91a286db5a3e4c435497c4592ec86f1a7d73

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
